route/build/src/mpi_utils.f90

```diff
@@ -975,7 +976,7 @@ SUBROUTINE shr_mpi_abort(message, ierr, comm)
     integer(i4b)                :: jerr
 
     write(iulog,*) trim(subName),trim(message),ierr
-    call flush(6)
+    call shr_sys_flush(6)
```

Comment:
shr_sys_flush usage...

Reply:
Talking with @nmizukami maybe the best solution would be to remove this flush statement.

Reply:
I've changed the flush statements in the shared src directory so they are in the FORTRAN 2003 keyword form:

flush(unit)